services/web/src/utils/user-agent.js

```diff
@@ -0,0 +1,47 @@
+export function parseUserAgent(ua) {
```

Comment:
uhoh this doesn't look good :)
is this needed in bedrock out of the box? if it really is it seems like the perfect thing for a lib to handle

Reply:
I mean there's a billion UA libs out there small to large, but more to the point why do we need to do this in the first place?

Reply:
this should be resolved, the usecase is just for clientside rendering. Accuracy is less important.